src/TransitionSystems/location_tuple.rs

```diff
@@ -44,6 +46,21 @@ impl LocationTuple {
             loc_type: location.get_location_type().clone(),
             left: None,
             right: None,
+            is_partial_location: false,
+        }
+    }
+    /// This method is used to a create partial [`LocationTuple`].
+    /// A partial [`LocationTuple`] means it has a [`LocationID`] that consists of atleast one [`LocationID::AnyLocation`].
+    /// A partial [`LocationTuple`] has not set field: `invariant`, `left` and `right` since a partial [`LocationTuple`]
+    /// covers more than one [`LocationTuple`], and therefore there is no specific `invariant`, `left` and `right`
+    pub fn create_partial_location(id: LocationID) -> Self {
```

Comment:
i dont know if important and there may be a good reasaon
but the other creation function "simple" does not take ID as input, is there a good reason why this one should?

Reply:
Simple finds the id in the field itself, so instead we just send it through parameter

Reply:
fair enough